ui/hooks/src/utils/entry-utils.ts

```diff
@@ -3,6 +3,7 @@ import { Comment } from '@akashaorg/typings/lib/sdk/graphql-types';
 import type { PostResultFragment } from '@akashaorg/typings/lib/sdk/graphql-operation-types';
 import type { BeamEntryData, ReflectEntryData } from '@akashaorg/typings/lib/ui';
 import { AkashaBeam, AkashaReflect } from '@akashaorg/typings/lib/sdk/graphql-types-new';
+import getSDK from '@akashaorg/awf-sdk';
 
 export const MEDIA_URL_PREFIX = 'CID:';
 export const TEXTILE_GATEWAY = 'https://hub.textile.io/ipfs/';
```

Comment:
unrelated to the pr changes, can we remove this?

Reply:
yes, they are no longer used